src/coins.ts

```diff
@@ -316,12 +316,14 @@ export const coins = CoinMap.fromCoins([
   erc20('zix', 'Zeex Token', 18, '0xf3c092ca8cd6d3d4ca004dc1d0f1fe8ccab53599', UnderlyingAsset.ZIX),
   erc20('zoom', 'CoinZoom', 18, '0x69cf3091c91eb72db05e45c76e58225177dea742', UnderlyingAsset.ZOOM),
   erc20('zrx', '0x Token', 18, '0xe41d2489571d322189246dafa5ebde1f4699f498', UnderlyingAsset.ZRX),
+  stellarToken('xlm:VELO-GC7GMEKN2P5LKGOVW55WGHMVQRPPRPHIRFMIC75Z6WPYPYR7B3Z3WEKH', 'Velo Token', 7, UnderlyingAsset['xlm:VELO-GC7GMEKN2P5LKGOVW55WGHMVQRPPRPHIRFMIC75Z6WPYPYR7B3Z3WEKH']),
```

Comment:
Can we use the prefix and suffix fields? The crappy thing with this formatting is that now all statics consumers are going to call the token "xlm:token-issuer", even in UI things. Trust admin tool for example will try to display that whole thing as the token name which is slightly ugly imo, it should just show "VELO-issuer" or maybe even just "VELO"